api/pkg/block/router.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020 The OpenSDS Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package block
+
+import (
+	"github.com/emicklei/go-restful"
+	"github.com/micro/go-micro/client"
+)
+
+//RegisterRouter - route request to appropriate method
+func RegisterRouter(ws *restful.WebService) {
+	handler := NewAPIService(client.DefaultClient)
+	ws.Route(ws.GET("/{tenanatId}/storages/{backendId}/volumes").To(handler.RouteVolumeList)).Doc("List all block devices")
```

Comment:
1. API path doesn't look clean /storages/{backendId} What is the relationship between these resources.
2. Typo for TenantId
3. Can we rename function name to ListVolume

Reply:
path is changed to backend/..
Corrected the type
Renamed the function